src/ptvsd/server/_remote.py

```diff
@@ -13,11 +14,17 @@
 from ptvsd.server._util import new_hidden_thread
 from ptvsd.server.pydevd_hooks import install
 from ptvsd.server.daemon import session_not_bound, DaemonClosedError
+from pydevd_file_utils import get_abs_path_real_path_and_base_from_file
 
 
 global_next_session = lambda: None
 
 
+def _get_debugger_root():
+    ptvsd_abspath = os.path.abspath(get_abs_path_real_path_and_base_from_file(ptvsd.__file__)[0])
```

Comment:
If we consider `set_dont_trace_start_end_patterns` as a public pydevd API, then I think it's best if it does the `NormPath` stuff internally - ptvsd should just be able to do `os.path.abspath(ptvsd.__file__)` here, and call it a day. 

There's one other catch, though. Look at the comment here - it also applies to this case, since user code might do a bunch of things before calling `enable_attach`, including chdir.

https://github.com/microsoft/ptvsd/blob/b29c459ff844709e455d2b3c39068244d58f9a7d/src/ptvsd/server/__main__.py#L46-L54

I'm thinking that perhaps it's best to do the same thing for ptvsd/__init__.py as well? Then everything can just use `ptvsd.__file__`, and it's always guaranteed to be absolute etc.

Or we can make some helper methods there, like `ptvsd.package_dir()`. But I don't really see the downside of just forcing `__file__` to be absolute - and then it's impossible to make a mistake. Coincidentally, it seems to be always absolute in py3, so this is just a py2 compat thing.